tools/BuildMonitor/source.extension.vsixmanifest

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-    <Metadata>
-        <Identity Id="BuildMonitor..0a528553-98c4-4948-a43c-bef2784df92c" Version="1.5" Language="en-US" Publisher="Microsoft" />
-        <DisplayName>BuildMonitor</DisplayName>
-        <Description xml:space="preserve">BuildMonitor watches the solution build process and reports WinObjC errors.</Description>
-    </Metadata>
-    <Installation>
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Community" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Premium" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Pro" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Ultimate" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Enterprise" />
-    </Installation>
-    <Dependencies>
-        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-    </Dependencies>
-    <Assets>
-        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:VsixSubPath="BM" Path="|BuildMonitor;PkgdefProjectOutputGroup|" />
-        <Asset Type="NativeVisualizer" d:Source="File" Path="Visualizers\ObjectiveC.natvis" />
-        <Asset Type="StepFilter" d:Source="File" Path="Visualizers\ObjectiveC.natstepfilter" />
-    </Assets>
-    <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-    </Prerequisites>
+  <Metadata>
+    <Identity Id="BuildMonitor..0a528553-98c4-4948-a43c-bef2784df92c" Version="1.6.1" Language="en-US" Publisher="Microsoft" />
```

Comment:
>1.6.1 [](start = 79, length = 5)

Why this version? Do we need to keep this in sync / auto rev in a more unified way?